docs/tools/vdb_table/data/datastaxastra.json

```diff
@@ -149,6 +149,15 @@
     "source_url": "",
     "comment": ""
   },
+  "index_types": {
+    "value": [
+      "FLAT",
+      "HNSW",
+      "FreshDiskANN"
+    ],
```

Comment:
Add the docs url here that states how JVector + Storage Attached Index (SAI) together form AstraDB's vector index.
https://docs.datastax.com/en/astra-db-serverless/get-started/concepts.html

Reply:
sure. done